pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -199,38 +200,38 @@ Here is a sample configuration file for the service:
 
 # Installation
 
-Now that you successfully created the config file (reffered to as `package-registry-options.json` from here on), you are ready to install the package registry. This can accomplished by following :
+Now that you have successfully created the config file (referred to as `package-registry-options.json` from here on), you are ready to install the package registry. This can accomplished by following :
 
 ```bash
 dcos package install package-registry --options=package-registry-options.json
 ```
 
-This would launch the marathon app for package-registry. This usually takes a couple of minutes. As soon as package-registry is healthy, you can add it as one of the package repositories in DC/OS. This can be done by:
+This would launch the Marathon app for `package-registry`. This usually takes a couple of minutes. As soon as `package-registry` is healthy, you can add it as one of the package repositories in DC/OS. This can be done by:
 
 ```bash
 # Change the repo name and URL if you need to customize
 dcos package repo add --index=0 "Registry" https://dcos-registry.marathon.l4lb.thisdcos.directory
 ```
 
-If you get errors in executing above command, wait for couple minutes (to account for the latency in package registry being healthy and its DNS entry being propogated to all master nodes) and try again.
+If you get errors in executing above command, wait for a couple of minutes (to account for the latency in `package-registry` being healthy and its DNS entry being propagated to all master nodes) and try again.
 
 # Using package registry
 
-After package registry is installed, you can start adding packages to it. There are two parts to this :
+After `package-registry` is installed, you can start adding packages to it. There are two parts to this :
 
 1. Building the package files (`.dcos` files)
-2. Uploading the packages to package-registry.
+2. Uploading the packages to `package-registry`.
 
 
 ## Building the packages
 
-Mesosphere hosts all its certified packages at https://downloads.mesosphere.com/universe/packages/packages.html If the packages you need are available there, you can download them and skip to the next section of uploading these .dcos files to your cluster. When a universe catalog package is under development and you want to test it before creating a pull request OR if you want to build a non certified (community) package, this section is useful.
+Mesosphere hosts all its certified packages at https://downloads.mesosphere.com/universe/packages/packages.html If the packages you need are available there, you can download them and skip to the next section of uploading these .dcos files to your cluster. When a {{ model.packageRepo }} package is under development and you want to test it before creating a pull request, or if you want to build a non certified (community) package, this section is useful.
```

Comment:
Use lowercase `i` in `if`

Reply:
Actually, that "If" is the beginning of a sentence so it should be capitalized. I am adding the period which is missing at the end of the previous sentence.